spec/presenters/two_factor_login_options_presenter_spec.rb

```diff
@@ -61,18 +61,95 @@
       )
   end
 
-  context 'with multiple webauthn configurations' do
-    let(:user) { create(:user) }
-    before(:each) do
-      create_list(:webauthn_configuration, 2, user: user)
-      user.webauthn_configurations.reload
+  describe '#options' do
+    let(:user) do
+      create(
+        :user,
+        :fully_registered,
+        :with_webauthn,
+        :with_webauthn_platform,
+        :with_phone,
+        :with_piv_or_cac,
+        :with_personal_key,
+        :with_backup_code,
+        :with_authentication_app,
+      )
+    end
+
+    subject(:options) { presenter.options }
+    let(:options_classes) { options.map(&:class) }
+
+    it 'returns classes for mfas associated with account' do
+      expect(options_classes).to contain_exactly(
```

Comment:
I think it's probably fair to say that order is significant on these, so I'll switch from `contain_exactly` (which doesn't consider order) to `eq [ ... ]`